index.html

```diff
@@ -368,15 +368,16 @@ <h1 id="fileName" class="no-zen"><span id="currentFileLocation" style="display:n
 					</li>
 					<li>
 						<select data-change="style" id="style-select" data-l10n-title="formatting">
-							<option value="" disabled selected data-l10n-id="format-block"></option>
-							<option value="p" data-l10n-id="paragraph"></option>
+							<option value="" disabled selected data-l10n-id="format-block"			style="font-size: 1.8rem;	font-weight: initial;	color: initial;"></option>
+							<option value="p" data-l10n-id="paragraph"								style="font-size: 1.8rem;	font-weight: initial;	color: initial;"></option>
```

Comment:
If you set your tab width to 4 it's aligned. I could changes it to spaces if you want, but that makes things a bit harder.